charts/aws-vpc-cni/values.yaml

```diff
@@ -43,6 +43,7 @@ nodeAgent:
     privileged: true
   enableCloudWatchLogs: "false"
   enablePolicyEventLogs: "false"
+  policyEventLogFileLocation: "/var/log/aws-routed-eni/network-policy-agent.log"
```

Comment:
Log file will have both NP agent logs & policy event logs and so I think it is better we use a generic name such as `networkPolicyAgentLogFileLocation`